pkg/alertmanager/alertstore/store.go

```diff
@@ -26,7 +26,7 @@ type AlertStore interface {
 	// GetAlertConfigs loads and returns the alertmanager configuration for given users.
 	// If any of the provided users has no configuration, then this function does not return an
 	// error but the returned configs will not include the missing users.
-	GetAlertConfigs(ctx context.Context, userIDs []string) (map[string]alertspb.AlertConfigDesc, error)
+	GetAlertConfigs(ctx context.Context, userIDs []string, fetchGrafanaConfig bool) (map[string]alertspb.AlertConfigDescs, error)
```

Comment:
Bool arguments like this don't sit right with me, they are confusing from the call site.

In this case, because the value is always the same (i.e. it's not per-tenant), then I would pass it in when the stores are created. This is also clearer, because it implies the value doesn't change.